frontend/src/components/dashboard/dashboard.tsx

```diff
@@ -38,7 +38,10 @@ const Dashboard: FC = () => {
   }
 
   const handleNewCourseModalToggle = (evt: React.MouseEvent | void): void => {
-    evt?.stopPropagation();
+    if (evt && evt.target) {
+      evt.stopPropagation();
+    }
```

Comment:
```
dispatch(action.addNewCourse).unwrap().then(onClose)
```
should be something like this

Reply:
@what1s1ove yes, we have it exactly like that. but we still pass the created course data to onClose function

Reply:
try wrapping it like this: then(()=>{onClose()})